src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DacPropertyAttributes/FieldTypeAttributesOnDacPropertyTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Threading.Tasks;
+using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.DacPropertyAttributes;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.DacPropertyAttributes
+{
+	public class FieldTypeAttributesOnDacPropertyTests : DiagnosticVerifier
+	{
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => new DacPropertyAttributesAnalyzer();
+
+		[Theory]
+		[EmbeddedFileData("DacWithMultipleFieldTypeAttributes.cs")]
+		public virtual void Property_With_Multiple_FieldTypeAttributes(string source) =>
```

Comment:
Recently we agreed to get rid of underscores as a word separator in test case names

Reply:
@dropsonic I thought we agree to use them to separate logical parts? Did we decide to not use them at all?

Reply:
@SENya1990 yes, but "Property" and "With" are not logical parts, they are words.
In this case, the whole test purpose is the one and only logical part: "PropertyWithMultipleFieldTypeAttributes".
The other part can be, for example, a place (`PXGraphInitializer`, `DacDeclaration`) or expected result, if it is unusual (e.g., `ShouldNotShowDiagnostic` or `ShouldThrowArgumentException`).

Reply:
ok